python/src/KrigingResult_doc.i.in

```diff
@@ -292,6 +292,33 @@ for each point in sampleX."
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::KrigingResult::operator()
+"Compute the expected Gaussian distribution on a new point / sample conditionally to the observed paths.
+
+Available usages:
+    KrigingResult(pt)
+
+    KrigingResult(sampleX)
+
+Parameters
+----------
+pt : sequence of float
+    The point :math:`\vect{x}` where the conditional distribution of the output has to be evaluated.
+sampleX : 2-d sequence of float
+     The sample :math:`(\vect{\xi}_1, \dots, \vect{\xi}_M)` where the conditional distribution of the output has to be evaluated (*M* can be equal to 1).
+
+Returns
+-------
+condDist : :class:`~openturns.Normal`
+    The conditional Gaussian distribution.
```

Comment:
OK, you added the word "Gaussian", but it is still vague in my opinion (see conversation with @regislebrun). What  is a "Gaussian" distribution? Is it a multivariate Normal distribution (the distribution of a Gaussian vector) or the distribution of a Gaussian process? Here, it is the latter, and this needs to be clearly stated. If you dislike my suggestion, do you have another equivalent formulation to propose?
```suggestion
 The distribution of the Gaussian process conditioned on the observations.
```

Reply:
In fact you are just focusing on the `Returns` section whereas section begins with 
```%feature("docstring") OT::KrigingResult::operator()
 "Compute the expected Gaussian distribution on a new point / sample conditionally to the observed paths.
```
It is useless to emphase again on the `conditioned on the observations` here

Reply:
Yes, you are right, I did not see that. But I also think "expected Gaussian distribution" does not make much sense. The conditioned Gaussian process is obtained as the expected value of the Gaussian process conditionally on the observations. However, its distribution is a "conditional distribution", not an "expected distribution" (can you define this term?). I understand that you're trying to avoid repetitions, but we need to use the same terminology as the literature, even if it makes the style clunky.

Reply:
Agreed with the terminology.
In that case you have to change this in many places (see conditional mean, conditional covariance, conditional variance...)